package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "hapi-react-views",
-  "version": "10.1.3",
+  "version": "11.0.0",
```

```suggestion
  "version": "10.1.3",
```

Let's revert this part since it'll happen when I publish to npm.